### PR TITLE
Convex and WBTC in Treasury Page

### PR DESCRIPTION
**Pull request by OleanjiKingCode**
# Adding Convex and wbtc in treasury page

## How should this be tested?

1. Before
image

2. After

image

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/1133

### CODE REVIEW

File: src/utils/treasury-utils.ts [hunk 1]
```diff
@@ -49,10 +49,16 @@ export const getTreasuryDetails = async () => {
     protocolId: 'apestake',
     id: config.treasuryAddress as string,
   }
+
+  const protocolDetailsPayloadn = {
```
Comment (Emmanueldmlr) [inline src/utils/treasury-utils.ts:53]: Rename the variable. It's confusing.

File: src/utils/treasury-utils.ts [hunk 2]
```diff
@@ -61,17 +67,22 @@ export const getTreasuryDetails = async () => {
     await fetchEndpointData(protocolDetailsPayload, '/api/protocols')
   ).portfolio_item_list[0].asset_token_list[0]
 
+  const contractProtocoldetailsn: LpTokenDetailsType[] = (
```
Comment (Emmanueldmlr) [inline src/utils/treasury-utils.ts:70]: Rename the variable. It's confusing.

File: src/utils/treasury-utils.ts [hunk 3]
```diff
@@ -95,7 +109,9 @@ export const getTreasuryDetails = async () => {
       })
     }
   })
+
   const allTreasureDetails = [...treasuryDetails, ...additionalTreasuryData]
+  console.log(allTreasureDetails)
```
Comment (Emmanueldmlr) [inline src/utils/treasury-utils.ts:114]: remove console.log

**Review by Emmanueldmlr — CHANGES_REQUESTED** [review]

File: src/utils/treasury-utils.ts [hunk 4]
```diff
@@ -50,7 +50,7 @@ export const getTreasuryDetails = async () => {
     id: config.treasuryAddress as string,
   }
 
-  const protocolDetailsPayloadn = {
+  const ConvexProtocolPayload = {
```
Comment (kesar) [inline src/utils/treasury-utils.ts:53]: why this is capital letter and everything else isnt?

**Review by kesar — CHANGES_REQUESTED** [review]

File: src/utils/treasury-utils.ts (hunk 4 above)
Comment (OleanjiKingCode) [reply src/utils/treasury-utils.ts]: its now fixed and all vars are in camelcase

**Pull request merged**